lib/DistributionProvider.pm

```diff
@@ -44,6 +45,7 @@ sub provide {
     return Distribution::Sle::12->new() if is_sle('12+');
     return Distribution::Sle::16Latest->new() if is_leap('16.0+');
     return Distribution::Opensuse::Leap::15->new() if is_leap('15.0+');
+    return Distribution::Opensuse::Leap::16Latest->new() if is_leap('16.0+');
```

Comment:
there is a mess here, line 46 is wrong, it is referring to SLE but the condition is for leap.
Sle::16latest doesn't exist yet, so I guess that is what you want to update.

Reply:
updated.